primitives/primproc/primitiveserver.cpp

```diff
@@ -696,13 +696,25 @@ void loadBlock (
                 i = fp->pread( &cmpHdrBuf[0], 0, 4096 * 3);
```

Comment:
4096 here and below is some random number or HDR_BUF_LEN?

Reply:
Seems like it's HDR_BUF_LEN, I'm not sure, why did author write 4096, instead of HDR_BUF_LEN.